alpharaw/ms_data_base.py

```diff
@@ -6,7 +6,14 @@
 
 class MSData_Base:
     """
-    The base data structure for MS Data, other MSData loader inherit
+    The base data structure for MS Data, other MSData loader inherit this class.
```

Comment:
mb be more specific on what "MS Data" is here .. raw data I guess?

Reply:
We support not only raw data but also standard format like mzml. MB MS spectrum data?